### DISCUSSION TITLE
How does the "News" feed populate?

### DISCUSSION THREAD

**Discussion by Vapok** [mastodon/mastodon#20537] [Q&A]
I run a fairly new Mastodon server for a niche community focused around a topical based server.  Think topic specific community.

We've got good Federation occuring, trends are populating successfully, Federated content and Local content are working as expected.

But the "News" pane is still completely empty and as a server admin, I see no way for me to influence or seed that view.

How do I activate that section?

**Comment by aureliendossantos** [comment]
I looked at the code the other day and it depends on the Open Graph metadata of the links shared by your federation. It has to be an article, have a title and a description, an so on.

Most press sites have those, so they can show up, but I even got a Github Gist once, cause apparently they use the article Open Graph type.
